frontend/src/components/pages/details/components/DetailsPreview/DetailsPreview.tsx

```diff
@@ -132,105 +133,192 @@ export const DetailsPreview: React.FC<DetailsPreviewProps> = ({
       </div>
       <div id="details_infoIcons" className="flex flex-wrap">
         {informations.difficulty && (
-          <RemoteIconInformation
-            iconUri={informations.difficulty.pictogramUri}
-            className={classNameInformation}
+          <ToolTip
+            toolTipText={intl.formatMessage({ id: 'tooltip.difficulty' })}
+            bottom="80%"
+            reverse={true}
```

Comment:
```suggestion
            reverse
```
When the attribute value is `true`, you can omit it :-)